<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_produit</class>
 <widget class="QMainWindow" name="gestion_produit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gestion_produit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>591</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-20</y>
      <width>501</width>
      <height>451</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>border-radius: 20px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(184, 166, 73);</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(184, 166, 73);</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>160</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(184, 166, 73);</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>280</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(184, 166, 73);</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="afficher">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>400</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 20px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(184, 166, 73);</string>
     </property>
     <property name="text">
      <string>Afficher</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ID_PRODUIT">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>id_produit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NOM">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IMAGE">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Image</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DISPONIBILITE">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Disponibilite</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DISCRIPTION">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MATIERE_PREMIERE">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>300</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Matiere_premiere</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CATEGORIE">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>360</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Categorie</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1200</width>
      <height>631</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/380386414_855923546066163_830367517448804430_n.png</pixmap>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>tableView</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
